src/webgpu/api/validation/render_pipeline/createRenderPipeline.spec.ts

```diff
@@ -203,8 +211,29 @@ g.test('at_least_one_color_state_is_required_for_complete_pipeline')
     t.doCreateRenderPipelineTest(isAsync, false, badDescriptor);
   });
 
-g.test('color_formats_must_be_renderable')
-  .desc(`TODO: review and add description; shorten name`)
+g.test('fragment_state,max_color_attachments_limit')
+  .desc(
+    `Tests that color state targets length must not be larger than device.limits.maxColorAttachments`
+  )
+  .params(u => u.combine('isAsync', [false, true]).combine('targetsLength', [1, 8, 16, 99]))
```

Comment:
test the minimal value that exceeds the limit, i.e. 9, to ensure the boundary is in exactly the right place. (no off by one errors). No need to test 1, 16, and 99 I think, just 8 and 9 is enough. (1 will inevitably be tested by tons of other tests anyway since it's so basic)